plugins/confirm.cpp

```diff
@@ -46,10 +42,8 @@ static map<string, conf_wrapper*> confirmations;
 string active_id;
 queue<string> cmds;
 
-// true when confirm is paused
-bool paused = false;
-// if set, confirm will unpause when this screen is no longer on the stack
-df::viewscreen *paused_screen = NULL;
+// TODO: move to confirmation instance with paused_focus etc
+ df::viewscreen *paused_screen = NULL;
```

Comment:
I _think_ this is no longer needed since we track if each confirmation is paused individually and they will each unpause during render if the focus string is no longer found. I commented this during my first pass at refactoring pause to use focus strings before implementing the per-confirmation pause stuff. So I've gone ahead and removed it for now.